### DISCUSSION TITLE
RuntimeError: Sizes of tensors must match except in dimension 1. Got 22 and 21 in dimension 3 (The offending index is 1)

### DISCUSSION THREAD

**Discussion by bibinwils** [qubvel/segmentation_models.pytorch#410] [Q&A]
I was trying with UNet with the example file given. I'm getting the following error. I thought it's because of the input dimensions tried to change the input to 224x224(crop) and no help by that. This is the error getting
RuntimeError: Sizes of tensors must match except in dimension 1. Got 22 and 21 in dimension 3 (The offending index is 1)

**Comment by kylelai112** [comment]
Have you solve this issue? 
I'm having the same issue when try to run validation. training is ok, It seem like interpolate function is causing the issue? 

![image](https://user-images.githubusercontent.com/75777822/132444740-c08914af-3220-477b-99c0-4bd4d264b143.png)

**Reply by kylelai112** [reply]
I solved the issue by resizing all the images size divisible to 32.
